src/App.js

```diff
@@ -1,28 +1,10 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
     <div className="App">
```

Comment:
- [ ] Kindly make your App.js component a class-based component. It's a requirement that these components be class-based components. Especially the Calculator and App components should be class-based for now 👍

Reply:
Thank you. Done now @AmaduKamara